libcontainer/rootfs_linux.go

```diff
@@ -152,14 +153,46 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 		}
 		return nil
 	case "tmpfs":
+		copyUp := m.Extensions&configs.EXT_COPYUP == configs.EXT_COPYUP
+		tmpDir := ""
```

Comment:
Not sure why `tmpDir` is global? is it because of the `defer`? I'd personally prefer if we put it in a closure, but that's just my 2c.